docs/ref/configuration.rst

```diff
@@ -129,6 +129,7 @@ An example configuration file looks like the following::
   [replication]
   slot = pgautofailover_standby
   maximum_backup_rate = 100M
+  backup_directory = /data/backup/node1.db.local.tld
```

Comment:
can use a more meaningful default here

Reply:
The default is `${pgdata}/../backup/${nodename}` in the code, and so in the sample configuration file too. Do you mean that the code should use another default value? If yes, which one?

Reply:
`node1.db.local.tld` looked non-trivial. 

I meant something like `/data/pg_autoctl/backup/nodedata.db` or something like that.

Reply:
Good catch, changed to `node1.db`.